client/XQDebugger/xqyDebug.ts

```diff
@@ -177,15 +178,22 @@ export class XqyDebugSession extends LoggingDebugSession {
 
     protected async launchRequest(response: DebugProtocol.LaunchResponse, args: LaunchRequestArguments): Promise<void> {
         logger.setup(Logger.LogLevel.Stop, false);
+        logger.setup(Logger.LogLevel.Verbose, false);
+        console.debug(JSON.stringify(args));
```

Comment:
Should this be left in? Could args have a password value in it?

Reply:
Good catch. Removed.